plugins/system/tour/tour.php

```diff
@@ -150,28 +150,28 @@ public function onBeforeCompileHead()
 		{
 			HTMLHelper::_(
 				'script',
-				Uri::root() . 'media/plg_system_tour/js/guide.js',
+				Uri::root() . 'build/media_source/plg_system_tour/js/guide.js',
```

Comment:
Shall this run only when system is built? Or shall it run on a real Joomla installation, not a build environment? In the latter case the URLs are wrong because the "build/media_source"  folder is not available.

Reply:
@richard67  So, do I have to change the position of the files to "media" from "build/media_source"?

Reply:
Here in this script yes. But in the source code they are in `build/media_source`. The subfolder structure below that is same as the subfolder structure in `media`. When you run an `npm install` or an `npm run build:js` (and if scss also `npm run build:css`) on a development environment, then the files from `build/media_source` and below will be compiled and minified into the same place in the `media` folder.

Reply:
okay